commands/service_init.go

```diff
@@ -36,22 +35,21 @@ func newServiceInitCmd(e ServiceExecutor) *serviceInitCmd {
 	
 To get more information, see the page [service file from the documentation](https://docs.mesg.com/guide/service/service-file.html)`,
 		Example: `mesg-core service init
-mesg-core service init --name NAME
-mesg-core service init --current`,
-		Args:    cobra.NoArgs,
+mesg-core service init ./PATH_TO_SERVICE
+mesg-core service init --template TEMPLATE_URL`,
```

Comment:
```suggestion
mesg-core service init --template TEMPLATE_URL ./PATH_TO_SERVICE`,
```

Reply:
The path is optional, that's why it is only show once.